src/Verilator.cpp

```diff
@@ -123,8 +123,10 @@ static void process() {
         std::exit(0);
     }
 
+    bool topIfacesSupported = v3Global.opt.lintOnly() && !v3Global.opt.hierarchical();
```

Comment:
In V3Options.h make a topIfacesSupported() with this equation, then can avoid all the edits to pass down the functions.

Reply:
Done.